test_package/functional/test.md

```diff
@@ -154,17 +152,6 @@ If you instead wrote the list in Markdown like this:
 1.  McHale
 1.  Parish
 
-or even:
```

Comment:
Why did you remove these? Does mistune 3.x not support numbered lists with non-sequential numbers?

Reply:
I now remember, I removed it because what mistune does now is start the list with the number of the first item, like this:

```
<ol start:"3">
<li> ..</li>
<\ol> 
```

When you have this markdown

```
3. sample
8. sample
2. sample
```

The sequence on the translated html is 3,4,5. 
Anywya, I put it back, as there is no need to remove this either, just changed the html tag accordingly.

Other than that, all other changes to result.xml were needed as mistune just renders them a bit differently, specially related to new line characters and formating, but th econtent is essentialy the same.